generators/Exercises/Hamming.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Generators.Input;
+
+namespace Generators.Exercises
+{
+    public class Hamming : Exercise
+    {
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            foreach (var canonicalDataCase in CanonicalData.Cases)
+            {
+                canonicalDataCase.ExceptionThrown = canonicalDataCase.Expected is long ? null : typeof(ArgumentException);
+            }
```

Comment:
@jpreese yes i could have cast to an `int` since I know the values are going to fit in the int range, but in this case I didn't think it was worth it given the only thing I'm using it for is to determine whether the test case expects an exception.

Thanks for the quick review!